### PR TITLE
ADD PANELS FILE AND MODIFY MAIN TO RUN SMOOTHLY

### PR DESCRIPTION
**Pull request by quagghyy**
Please read

### CODE REVIEW

File: App.java
Comment (quagghyy) [inline App.java:1]: Add a space for easy reading code

**Comment by tunsimp** [comment]
what did you do to run smoothly? pls give more description and remember to leave comment for reading code easier. 👍

**Comment by tunsimp** [comment]
and also be specific of what you did, which classes, methods you worked with, what do they do

**Comment by quagghyy** [comment]
> what did you do to run smoothly? pls give more description and remember to leave comment for reading code easier. 👍
yes i will comment right now

File: BoardPanel.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import javax.swing.*;
+
+public class BoardPanel extends JPanel {
+    private MineSweeper mineSweeper;
+    private int numRows;
+    private int numCols;
+    private MineTile[][] board;
+    private ArrayList<MineTile> mineList;
+
+    public BoardPanel(MineSweeper mineSweeper, int numRows, int numCols, MineTile[][] board, ArrayList<MineTile> mineList) {
```
Comment (quagghyy) [inline BoardPanel.java:13]: Make a constructor that take all the variables inside it to initialize the Board Panel

Comment (quagghyy) [inline BoardPanel.java:13]: Make a constructor that take all the variables inside it to initialize the Board Panel

**Review by quagghyy — COMMENTED** [review]
This is Text Panel Class, i created this for encapsulation apply
- 1st is a constructor that create a new Text Panel, in this class just simple customization with the code, like set font, color and alignment of the text
- setTextLabel is a method to set the Text to the panel (Set the text Minesweeper, Game Overs and Mine Cleared)
- getTextLabel is to get the the text of the panel (no usage yet but still create to make it enough for set get method)

**Pull request merged**